### DISCUSSION TITLE
What is the latest version of ROCm that supports WSL2?

### DISCUSSION THREAD

**Discussion by alstpwns** [ROCm/ROCm#3895] [Q&A] [answered]
hello!

I recently read an article about ROCm 6.2.3.

["New AMD ROCm™ 6.2 for Radeon GPUs Delivers Performance & Compatibility with Models and Frameworks"](https://community.amd.com/t5/ai/new-amd-rocm-6-2-for-radeon-gpus-delivers-performance-amp/ba-p/715854)

![image](https://github.com/user-attachments/assets/a80bf7b6-5848-4cc9-8b94-66f326bad78c)

There it says that WSL2 with ROCm 6.2.1 support will be released soon, but the [official documentation](https://rocm.docs.amd.com/projects/radeon/en/latest/docs/compatibility/wsl/wsl_compatibility.html) seems to say that 6.2.3 is already supported.

![image](https://github.com/user-attachments/assets/86aa6e99-74b9-423c-8c21-df70375699fd)

As a beginner approaching Radeon AI applications for the first time, I'm confused, could you please clarify?

**Comment by schung-amd** [comment, marked as answer]
Hi @alstpwns, sorry for the confusion. The support matrix was updated erroneously and has been corrected to 6.1.3. You can find instructions for installing ROCm on WSL [here](https://rocm.docs.amd.com/projects/radeon/en/latest/docs/install/wsl/install-radeon.html), and these instructions should be updated when a new version comes out. One thing to note is that those instructions specify Adrenalin 24.6.1 while the compatibility matrix specifies 24.10.21.01; I suspect either version will work for ROCm 6.1.3.
